AppCenterDataStorage/AppCenterDataStorageTests/MSTokenTests.m

```diff
@@ -0,0 +1,24 @@
+
+#import "MSTestFrameworks.h"
+#import <Foundation/Foundation.h>
```

Comment:
The convention is place `#import <Foundation/Foundation.h>` before any local imports like `#import "MSTestFrameworks.h"`.